build/distributions/CentOS/6/x86_64/DVD/ks/dvd/auto.cfg

```diff
@@ -255,18 +275,18 @@ gpgkey=file:///var/www/yum/SIMP/GPGKEYS/RPM-GPG-KEY-puppet
     file:///var/www/yum/SIMP/GPGKEYS/RPM-GPG-KEY-grafana
     file:///var/www/yum/SIMP/GPGKEYS/RPM-GPG-KEY-PGDG-96
     file:///var/www/yum/SIMP/GPGKEYS/RPM-GPG-KEY-EPEL-$majrhversion
-    file:///var/www/yum/SIMP/GPGKEYS/RPM-GPG-KEY-CentOS-$majrhversion
-    $extrakey
+$flocal_gpg_file_lines
```

Comment:
To absolutely ensure the spacing is correct (which yum requires), I would add all the keys into the array (grafana, PDDG, EPEL...)